container-engine-lib/lib/backend_impls/kubernetes/object_attributes_provider/enclave_object_attributes_provider.go

```diff
@@ -354,7 +356,14 @@ func (provider *kubernetesEnclaveObjectAttributesProviderImpl) getLabelsForEncla
 	if err != nil {
 		return nil, stacktrace.Propagate(err, "An error occurred creating a Kubernetes label value from UUID string '%v'", uuid)
 	}
+	shortUuidStr := uuid_generator.ShortenedUUIDString(uuid)
+	shortUuidLabelValue, err := kubernetes_label_value.CreateNewKubernetesLabelValue(shortUuidStr)
+	if err != nil {
+		return nil, stacktrace.Propagate(err, "An error occurred creating a short GUID Kubernetes label value from GUID string '%v'", uuid)
+	}
 	labels[kubernetes_label_key.GUIDKubernetesLabelKey] = uuidLabelValue
+	labels[kubernetes_label_key.LogsServiceUUIDKubernetesLabelKey] = uuidLabelValue
```

Comment:
Cool, I like that you're storing both using the same logic here that goes long -> short (rather than trying to shorten on the consumer side, which would mean there's shortening logic drift between generation & consumption)